cmake/macos/xcode.cmake

```diff
@@ -58,7 +67,7 @@ set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_INJECT_BASE_ENTITLEMENTS[variant=RelWithDebI
 set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_INJECT_BASE_ENTITLEMENTS[variant=MinSizeRel] NO)
 
 # Use Swift version 5.0 by default
-set(CMAKE_XCODE_ATTRIBUTE_SWIFT_VERSION 5.0)
+set(CMAKE_XCODE_ATTRIBUTE_SWIFT_VERSION 5)
```

Comment:
Is this somehow different from 5.0? Wondering if the comment is still accurate.

Reply:
That's a mistake - `5.0` is the value accepted by Xcode, when specifying `5` it's treated as a custom (and ignored) setting.